packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -342,25 +342,40 @@ class _CupertinoDatePickerDateTimeState extends State<CupertinoDatePicker> {
   int textDirectionFactor;
   CupertinoLocalizations localizations;
 
-  // Alignment based on text direction. The variable name is self descriptive,
-  // however, when text direction is rtl, alignment is reversed.
+  /// Alignment based on text direction. The variable name is self descriptive,
```

Comment:
This is just a code comment. When it's a public facing API doc, we'll use ///. But if it's just leaving breadcrumbs for future maintainers rather than a doc for users, we'll leave it in // form.